src/main/java/tasks/HomeTask.java

```diff
@@ -0,0 +1,38 @@
+package tasks;
+
+import java.util.*;
+import java.util.function.Function;
+import java.util.stream.Collectors;
+
+public class HomeTask {
+
+    public static void main(String[] args) {
+        String[] words ={"мама", "тато", "їж їжак желе", "папа"};
+        Set<Character> uniqueChars = findUniqueCharacters(words);
+        System.out.println(uniqueChars);
+    }
+
+    public static Set<Character> findUniqueCharacters(String[] words) {
+        List<String> wordList = Arrays.asList(words);
+        List<String> filteredWords = wordList.stream()
+                .filter(word -> countEvenOccurrences(word) == 2)
+                .limit(2)
+                .collect(Collectors.toList());
+
+        Set<Character> uniqueChars = filteredWords.stream()
+                .flatMap(word -> word.chars().mapToObj(ch -> (char) ch))
+                .collect(Collectors.toSet());
+
+        return uniqueChars;
+    }
+
+    public static long countEvenOccurrences(String word) {
```

Comment:
This method just returns count of even number of characters. It does not make sense